docs/developers/plugin-development.md

```diff
@@ -38,33 +38,46 @@ The only requirement for the `trial` method is that it calls `jsPsych.finishTria
 
 ### static info
 
-The plugin's `info` property is an object with a `name` and `parameters` property. 
+The plugin's `info` property is an object with a `name`, `version`, `parameters`, and `data` property. 
 
 ```js
 const info = {
   name: 'my-awesome-plugin',
-  parameters: { }
+  version: version,
+  parameters: { },
+  data: { }
 }
 ```
 
+The `version` field should imported from the package.json file by including an import statement in the top of the index.ts file. This lets the `version` field be automatically updated with each changeset. 
```

Comment:
We should give an example of how to do this. We should also tell users what to do if they are not using a build environment and just writing a plain JS file. (in that case they can manually enter a version string)

Reply:
Addressed with short description.